### ISSUE TITLE
Wallet not opening on MIUI 10, xiaomi pie 9

### ISSUE THREAD

**Issue by cryptodadach** [ACINQ/eclair-mobile#139]
After the update the wallet is not opening on my phone, and I need the funds soon. Can you check?  Can I revert to the previous version?

**Comment by cryptodadach** [comment]
had to use the previous version of the .apk, and that one worked..

**Comment by dpad85** [comment]
What do you mean with "not opening on my phone"? Do you have any specific error message?

**Comment by cryptodadach** [comment]
No, just says please wait, and when I recovered the seed, it says reading
seed, but it is stuck in both cases. As soon as I installed the prior
version, it worked again.

My phone is Xiaomi Mi Mix 2s, and all prior versions worked without a
problem.

Thank you for replying,
Davor

**Comment by cryptodadach** [comment]
It just sits like this forever.

**Comment by Falsen** [comment]
This happend to me too when I tried to import my wallet to my new OnePlus 6T.

I checked logcat, and I didn't get much really but:

> W/fr.acinq.eclair.wallet.mainnet2: type=1400 audit(0.0:312427): avc: denied { read } for comm=45474C20496E6974 name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=15853 scontext=u:r:untrusted_app:s0:c236,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
> W/fr.acinq.eclair.wallet.mainnet2: type=1400 audit(0.0:312428): avc: denied { read } for comm=45474C20496E6974 name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=15853 scontext=u:r:untrusted_app:s0:c236,c256,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
